docs/root/intro/version_history.rst

```diff
@@ -72,6 +72,7 @@ Version history
 * tls: added support for :ref:`client-side session resumption <envoy_api_field_auth.UpstreamTlsContext.max_session_keys>`.
 * tls: added support for CRLs in :ref:`trusted_ca <envoy_api_field_auth.CertificateValidationContext.trusted_ca>`.
 * tls: added support for :ref:`password encrypted private keys <envoy_api_field_auth.TlsCertificate.password>`.
+* tls: added ssl.versions.<version> to track TLS versions in use
```

Comment:
nit: end with a period.

Reply:
Also, I'd add a ref to `config_listener_stats`.

Reply:
Done.